lib/shopify_transporter/exporters/exporter.rb

```diff
@@ -68,9 +69,9 @@ def ensure_config_has_required_keys
         ]
 
         product_required_keys = [
-          %w(export_configuration database hostname),
-          %w(export_configuration database username),
-          %w(export_configuration database name),
+          %w(export_configuration database host),
+          %w(export_configuration database user),
+          %w(export_configuration database database),
```

Comment:
Minor: I prefer leaving the last one as `name`, or maybe `database_name` because having a nested `database` key for the `database` hash seems odd.

Reply:
Fair, I just wanted to map it directly to the values a database expects (specifically mysql). It's been way less confusing that way, IME.

Reply:
i had this exact conversation in my head when i wrote this, i dislike `database database` as well personally. but i really dont care much